app/logic/emailHandler.py

```diff
@@ -158,6 +156,11 @@ def build_email(self):
 
     def send_email(self):
         template_id, subject, body = self.build_email()
+        if len(self.program_ids) == 1:
+            self.reply_to = self.program_ids[0].emailReplyTo
+            sender = (self.program_ids[0].emailSenderName, self.program_ids[0].emailReplyTo)
+        else:
+            sender = ("Sandesh", 'bramsayr@gmail.com')

```

Comment:
We need to make sure the defaults are set if the program doesn't have a sender name or reply to set.